integration-tests/src/test/java/oracle/weblogic/kubernetes/ItClusterResourceScaling.java

```diff
@@ -0,0 +1,527 @@
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain 2");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+    createTestRepoSecret(domainNamespace);
+  }
+
+  /**
+   * Install the Operator successfully.
+   * Create domain and verify the domain is started
+   * Verify cluster can be scale via Rest Api
+   * Verify expected http responce in case of :
+   * 1. Provided invalid auth token
+   * 2. Provided invalid domainuid
+   * 3. Provided invalid clustername
+   * 4. Provided above max replica
+   * 5. Missed header
+   * 6 Missed auth header
+   */
+  @Test
+  @DisplayName("Create domain, managed by operator and domain1, perfom scaling operation via REST  "
+      + "verify scaling operation generate expected code in case of negative scenarios: bad auth"
+      + "invalid domainuid, invalid clustername, invalid replica count, invalid request header, invalid auth header")
+  void testScaleClusterViaRestApi() {
+    HelmParams opHelmParams = null;
+    HelmParams op1HelmParams = new HelmParams().releaseName(OPERATOR_RELEASE_NAME)
+        .namespace(opNamespace)
+        .chartDir(OPERATOR_CHART_DIR);
+    String opServiceAccount = opNamespace + "-sa";
+    boolean badAuthFailed = false;
+    boolean invalidDomainUidFailed = false;
+    boolean invalidClusterNameFailed = false;
+    boolean invalidReplicaNumberFailed = false;
+    boolean invalidHeaderFailed = false;
+    boolean invalidAuthHeaderFailed = false;
+
+    StringBuffer failedNegativeTestCases = new StringBuffer();
+    String negativeTestCase1 = "Bad authentication";
+    String negativeTestCase2 = "Invalid domainUid";
+    String negativeTestCase3 = "Invalid clusterName";
+    String negativeTestCase4 = "Invalid replicaNumber";
+    String negativeTestCase5 = "Invalid request header";
+    String negativeTestCase6 = "Invalid auth header";
+    try {
+      // install operator
+      opHelmParams = installAndVerifyOperator(opNamespace, opServiceAccount, true,
+          0, op1HelmParams, domainNamespace).getHelmParams();
+      assertNotNull(opHelmParams, "Can't install operator");
+
+      int externalRestHttpsPort = getServiceNodePort(opNamespace, "external-weblogic-operator-svc");
+      assertNotEquals(-1, externalRestHttpsPort,
+          "Could not get the Operator external service node port");
+      logger.info("externalRestHttpsPort {0}", externalRestHttpsPort);
+      if (!isDomainRunning) {
+        logger.info("Installing and verifying domain");
+        assertTrue(createVerifyDomain(domainNamespace, domainUid),
+            "can't start or verify domain in namespace " + domainNamespace);
+        isDomainRunning = true;
+      }
+
+      // scale domain
+      int replicaCountdomain = 2;
```

Comment:
change replicaCountdomain -> replicaCountCluster since the replicaCount is for cluster.

Reply:
done